api/src/pcapi/core/offerers/api.py

```diff
@@ -1546,6 +1547,47 @@ def _cancel_collective_bookings_on_offerer_closure(offerer_id: int, author_id: i
     db.session.flush()
 
 
+def handle_closed_offerer(offerer: offerers_models.Offerer, closure_date: date | None) -> None:
+    action_kwargs: dict[str, typing.Any] = {
+        "comment": "L'entité juridique est détectée comme fermée "
+        + (closure_date.strftime("le %d/%m/%Y ") if closure_date else "")
+        + "via l'API Entreprise (données INSEE)"
```

Comment:
Dans le cas d'une fermeture par la commande `check_closed_offerers`, c'est directement par l'API Sirene de l'INSEE, donc je te propose plus générique : 
```suggestion
        + "au répertoire Sirene (INSEE)"
```